intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -40,6 +40,494 @@ \chapter{Running batch jobs}
+\lstinline|module restore|:
+
+\begin{prompt}
+%\shellcmd{module restore \emph{my-collection}}%
+\end{prompt}
+
+You can get a list of all your saved collections with the \lstinline|module savelist|
+command:
+
+\begin{prompt}
+%\shellcmd{module savelistr}%
+Named collection list (For LMOD_SYSTEM_NAME = "CO7-sandybridge"):
+  1) my-collection
+\end{prompt}
+
+To get a list of all modules a collection will load, you can use the \lstinline|module describe|
+command:
+\begin{prompt}
+%\shellcmd{module describe \emph{my-collection}}%
+1) example/1.2.3                                          6) imkl/11.3.3.210-iimpi-2016b
+2) GCCcore/5.4.0                                          7) intel/2016b
+3) icc/2016.3.210-GCC-5.4.0-2.26                          8) examplelib/1.2-intel-2016b
+4) ifort/2016.3.210-GCC-5.4.0-2.26                        9) secondexample/2.7-intel-2016b
+5) impi/5.1.3.181-iccifort-2016.3.210-GCC-5.4.0-2.26
+\end{prompt}
+\fi
+
+To remove a collection, remove the corresponding file in \lstinline|$HOME/.lmod.d|:
+
+\begin{prompt}
+%\shellcmd{rm \$HOME/.lmod.d/my-collection}%
+\end{prompt}
+
+\ifusinglmod
+\subsection{Getting module details}
+
+To see how a module would change the environment, you can use the \lstinline|module show|
+or \lstinline|ml show| command:
+
+\ifgent
+\begin{prompt}
+%\shellcmd{module show GSL/2.4-GCCcore-6.4.0}
+help([[
+Description
+===========
+The GNU Scientific Library (GSL) is a numerical library for C and C++
+ programmers.  The library provides a wide range of mathematical routines
+ such as random number generators, special functions and least-squares fitting.
+
+
+More information
+================
+ - Homepage: http://www.gnu.org/software/gsl/
+]])
+whatis("Description:
+ The GNU Scientific Library (GSL) is a numerical library for C and C++
+ programmers.  The library provides a wide range of mathematical routines
+ such as random number generators, special functions and least-squares fitting.
+")
+whatis("Homepage: http://www.gnu.org/software/gsl/")
+conflict("GSL")
+load("GCCcore/6.4.0")
+prepend_path("ACLOCAL_PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/share/aclocal")
+prepend_path("CPATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/include")
+prepend_path("LD_LIBRARY_PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/lib")
+prepend_path("LIBRARY_PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/lib")
+prepend_path("MANPATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/share/man")
+prepend_path("PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/bin")
+prepend_path("PKG_CONFIG_PATH","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/lib/pkgconfig")
+setenv("EBROOTGSL","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0")
+setenv("EBVERSIONGSL","2.4")
+setenv("EBDEVELGSL","/apps/gent/CO7/sandybridge/software/GSL/2.4-GCCcore-6.4.0/easybuild/GSL-2.4-GCCcore-6.4.0-easybuild-devel")
```

Comment:
I would i) use the output of the `module show Python/...` so you can point out that `Python` is usually installed with additional extensions, ii) clean up the output a bit to only show what's relevant.

So, only include one single `load` statement (e.g. for the toolchain), replace the others with a single `...`.
Also only include a single `prepend_path` statement, like this: `prepend_path(...)`.

When showing the output, you still need to discriminate between Lmod & not.
Log in to Leuven to check output without Lmod (and also trim it down).